client/lib/menu-data/menu-data.js

```diff
@@ -349,10 +366,6 @@ MenuData.prototype.interceptLoadForHomepageLink = function ( menu ) {
 	return menu;
 };
 
-/**
- * @param {object} menu
- * @param {function} [optional] callback
- */
```

Comment:
I decided remove them because it doesn't offer additional information in itself. The information is right there an implicitly way. Also it triggers an eslint warning because it doesn't have description for `menu` and `callback` parameters.